### PR TITLE
Reduced bundled size, Added second source map file to bundle, Added npm instructions to README.md

### PR DESCRIPTION
**Pull request by ctwomblyamzn**
*Description of changes:*
    * Reduced chatjs' bundle size by separating the source map to a separate file. 
    * Added npm  install / include instructions to the readme. 
    * Bumped ChatJS' version.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,11 +30,19 @@ Import ChatJS before the AWS SDK, and ensure the AWS SDK version you are using c
 #### A note for Scenarios 2 and 4.
 When using the SDK and ChatJS, you may remove the SDK from ChatJS to ensure lack of import conflicts. However, this should not be relevant if the order in which you are importing these libraries is the order reflected above.
 
-### Downloading ChatJS
+### Using ChatJS from npm
+
+`npm install amazon-connect-chatjs -save`
```
Comment (haochi-aws) [inline README.md:35]: double `--` for `save`? also the doc is no longer showing `--save` as an option https://docs.npmjs.com/cli/install

btw https://stackoverflow.com/questions/36022926/what-do-the-save-flags-do-with-npm-install#comment79204893_36023156 says `--save` is the default behavior now.

File: README.md [hunk 2]
```diff
@@ -30,11 +30,19 @@ Import ChatJS before the AWS SDK, and ensure the AWS SDK version you are using c
...
+
+# Importing using npm and ES6
+`import amazon-connect-chatjs`
```
Comment (haochi-aws) [inline README.md:38]: quotes around `amazon-connect-chatjs`?

File: README.md (hunk 1 above)
Comment (ctwomblyamzn) [reply README.md:35]: Got it, I'll get rid of it

File: README.md (hunk 2 above)
Comment (ctwomblyamzn) [reply README.md]: Fixed

**Pull request merged**